cluster-provision/gocli/opts/docker-proxy/docker-proxy.go

```diff
@@ -0,0 +1,48 @@
+package dockerproxy
+
+import (
+	"embed"
+	"strings"
+
+	"kubevirt.io/kubevirtci/cluster-provision/gocli/pkg/libssh"
+)
+
+//go:embed conf/*
+var f embed.FS
+
+type DockerProxyOpt struct {
+	proxy     string
+	sshClient libssh.Client
+}
+
+func NewDockerProxyOpt(sc libssh.Client, proxy string) *DockerProxyOpt {
+	return &DockerProxyOpt{
+		proxy:     proxy,
+		sshClient: sc,
+	}
+}
+
+func (o *DockerProxyOpt) Exec() error {
+	override, err := f.ReadFile("conf/override.conf")
+	if err != nil {
+		return err
+	}
+	script := strings.ReplaceAll(string(override), "$PROXY", o.proxy)
```

Comment:
how about using text.template here instead?

Reply:
i think for what i want its too complicated
since i would have to export the field i want to replace then also create a resulting string builder and provide a pointer to it in template.Execute

unless there's a simpler way that i don't know about ?